backend/app/usecases/movies/create.rb

```diff
@@ -0,0 +1,26 @@
+class Movies::Create
+  def initialize(params:, movie: Movie, repository: MoviesRepository)
+    @params = params
+    @movie = movie
+    @repository = repository
+  end
+
+  def call
```

Comment:
Please change the naming here, too
```suggestion
  def call!
```